src/app/results/results.component.html

```diff
@@ -22,9 +22,12 @@
             {{message}}
         </div>
         <div class="text-result" *ngIf="Display('all')">
-            <div *ngFor="let item of items$|async" class="result">
-                <div class="title">
+            <div *ngFor="let item of items$|async let i = index; trackBy: trackHero" class="result">
+                <div class="title" (mouseover)="overTitle()" (mouseleave)="overTitle()">
                     <a href="{{item.link}}">{{item.title}}</a>
+                    <div class="box">
+                        <iframe width="400px" height="400px" [src]="myUrlList[i]"></iframe>
```

Comment:
@nikhilrayaprolu Any suggestions? As this was the only possible way to do it. I was implementing this feature dynamically but there were some security trust issues due to which preview was not been shown.